src/main/app/src/app/taken/taak-view/taak-view.component.ts

```diff
@@ -341,15 +341,6 @@ export class TaakViewComponent
     }
   }
 
-  partialEditTaak(value: string, field: string): void {
```

Comment:
This function was never called, so I removed it. This in turn was the only code resulting in a call to takenService.update, so it allowed for a method on te back end to be made private. See other comment.